src/Dofs/ConstraintHandler.jl

```diff
@@ -61,6 +61,78 @@ function __to_components(c)
     return components
 end
 
+@doc raw"""
+    L2ProjectedDirichlet(field_name::Symbol, facets::AbstractSet{FacetIndex}, f::Function; qr_order = -1)
```

Comment:
I assume we could also apply the H(curl)-conforming variant to ridges? If so, should we open a good-first-issue on this for potential contributors?

Reply:
I don't know if that arises from some derivations (I thought so initially, but doesn't seem like it based on what I found when writing the following: https://ferrite-fem.github.io/Ferrite.jl/previews/PR798/tutorials/maxwell_good_bad_ugly/#Theoretical-background)